ejb/src/main/java/org/wildfly/httpclient/ejb/HttpInvocationHandler.java

```diff
@@ -140,14 +141,20 @@ public SocketAddress getLocalAddress() {
                 }
 
                 @Override
-                public Resolved getRequestContent(ClassLoader classLoader) throws IOException, ClassNotFoundException {
+                public Resolved getRequestContent(final ClassLoader classLoader) throws IOException, ClassNotFoundException {
 
                     Object[] methodParams = new Object[parameterTypeNames.length];
                     final Class<?> view = Class.forName(viewName, false, classLoader);
                     final MarshallingConfiguration marshallingConfiguration = new MarshallingConfiguration();
                     marshallingConfiguration.setObjectTable(HttpProtocolV1ObjectTable.INSTANCE);
                     marshallingConfiguration.setVersion(2);
-                    Unmarshaller unmarshaller = HttpServerHelper.RIVER_MARSHALLER_FACTORY.createUnmarshaller(marshallingConfiguration);
+                    marshallingConfiguration.setClassResolver(new AbstractClassResolver() {
```

Comment:
Hi Jaikiran, you could avoid adding another class here by using org.jboss.marshalling.SimpleClassResolver instead.

Reply:
@dmlloyd, Makes sense. Updated the PR accordingly.